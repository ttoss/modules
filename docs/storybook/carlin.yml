# Skip deploy for nom production environments.
skipDeploy: true

environments:
  Production:
    aliases:
      - storybook-modules.ttoss.dev
    cloudfront: true
    csp: false
    skipDeploy: false
    stackName: TtossModulesDocsStorybook
