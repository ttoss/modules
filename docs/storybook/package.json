{
  "name": "@docs/storybook",
  "version": "0.0.0",
  "private": true,
  "author": "ttoss",
  "scripts": {
    "storybook": "start-storybook -p 6006",
    "build:after": "build-storybook --quiet",
    "start": "yarn run storybook",
    "dev": "yarn run storybook"
  }
}
