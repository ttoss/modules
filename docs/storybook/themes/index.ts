import { theme as Siflor } from './siflor';
import { theme as Triangulos } from './triangulos';

import type { Theme } from '@ttoss/ui';

export const themes: { [key: string]: Theme } = {
  Siflor,
  Triangulos,
};
