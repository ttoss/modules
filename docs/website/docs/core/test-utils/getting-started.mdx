---
hide_title: true
title: Getting Started
---

import Readme, { toc as ReadmeTOC } from '@ttoss/test-utils/README.md';

<Readme />

export const toc = ReadmeTOC;
