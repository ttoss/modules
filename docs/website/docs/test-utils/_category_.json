{
  "label": "Test Utils",
  "position": 1
}
