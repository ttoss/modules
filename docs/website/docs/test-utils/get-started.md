---
sidebar_position: 1
title: Getting Started
---

import Readme, { toc as ReadmeTOC } from "@ttoss/test-utils/README.md"

<Readme />

export const toc = ReadmeTOC;
