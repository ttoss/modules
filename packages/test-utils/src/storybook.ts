export { default as initStoryshots } from '@storybook/addon-storyshots';
/**
 * This package needs `@storybook/client-api` package.
 */
export * from '@storybook/testing-react';
