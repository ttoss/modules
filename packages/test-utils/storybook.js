module.exports = require('./dist/storybook');
