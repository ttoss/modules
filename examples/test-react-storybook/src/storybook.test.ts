import { initStoryshots } from '@ttoss/test-utils/storybook';

initStoryshots();
