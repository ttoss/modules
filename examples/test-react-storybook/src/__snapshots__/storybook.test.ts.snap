// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Component Example 1`] = `
<div>
  <span>
    StorybookDecorator
  </span>
  <div>
    <span>
      oi
    </span>
    <span>
      0
    </span>
    <button
      onClick={[Function]}
    >
      Increment
    </button>
  </div>
</div>
`;
